sql/src/main/java/org/apache/druid/sql/http/SqlQuery.java

```diff
@@ -85,13 +110,14 @@ public boolean equals(final Object o)
     return header == sqlQuery.header &&
            Objects.equals(query, sqlQuery.query) &&
            resultFormat == sqlQuery.resultFormat &&
-           Objects.equals(context, sqlQuery.context);
+           Objects.equals(context, sqlQuery.context) &&
```

Comment:
Suggest adding an `EqualsVerifier` unit test for this

Reply:
added